src/entity.ts

```diff
@@ -163,6 +195,14 @@ export class Entity {
 		else
 			properties = property;
 
+		if (!this._context) {
+			const context = new InitializationContext(true);
```

Comment:
Am I misreading this, or does this mean when you call `update` the context has "newDocument"=true? I don't know if it matters though...

Reply:
You're reading it right. It pretty much just ensures any "owned" objects created via `update()` are considered to be "new".

```ts
entry.update({
  Section: {
    Field1: 'x',
    Field2: 'y'
  }
});

console.log(entry.Section.meta.isNew); // true
```

Reply:
Ah, that makes sense